reliability-engineering/terraform/modules/concourse-monitoring/files/dashboards/concourse-slo.json

```diff
@@ -0,0 +1,417 @@
+              "type": 1,
+              "value": ""
+            }
+          ],
+          "thresholds": {
+            "mode": "percentage",
+            "steps": [
+              {
+                "color": "dark-red",
+                "value": null
+              },
+              {
+                "color": "dark-yellow",
+                "value": 99
+              },
+              {
+                "color": "dark-green",
+                "value": 99.9
+              }
+            ]
+          },
+          "unit": "percentunit"
+        },
+        "overrides": []
+      },
+      "gridPos": {
+        "h": 8,
+        "w": 6,
+        "x": 0,
+        "y": 0
+      },
+      "id": 6,
+      "interval": "",
+      "options": {
+        "colorMode": "value",
+        "graphMode": "area",
+        "justifyMode": "auto",
+        "orientation": "auto",
+        "reduceOptions": {
+          "calcs": [
+            "mean"
+          ],
+          "fields": "/^errorRatio$/",
+          "values": false
+        }
+      },
+      "pluginVersion": "7.0.5",
+      "targets": [
+        {
+          "alias": "",
+          "apiMode": "Metrics",
+          "dimensions": {
+            "LoadBalancer": "app/cd-concourse-web/ba5029fcd9b9cde2"
+          },
+          "expression": "",
+          "hide": true,
+          "id": "status5xx",
+          "matchExact": true,
+          "metricName": "HTTPCode_ELB_5XX_Count",
+          "namespace": "AWS/ApplicationELB",
+          "period": "300",
+          "refId": "A",
+          "region": "eu-west-2",
+          "statistics": [
+            "Sum"
+          ]
+        },
+        {
+          "alias": "",
+          "dimensions": {
+            "LoadBalancer": "app/cd-concourse-web/ba5029fcd9b9cde2"
+          },
+          "expression": "",
+          "hide": true,
+          "id": "status4xx",
+          "matchExact": true,
+          "metricName": "HTTPCode_ELB_4XX_Count",
+          "namespace": "AWS/ApplicationELB",
+          "period": "300",
+          "refId": "B",
+          "region": "default",
+          "statistics": [
+            "Sum"
+          ]
+        },
+        {
+          "alias": "",
+          "dimensions": {
+            "LoadBalancer": "app/cd-concourse-web/ba5029fcd9b9cde2"
+          },
+          "expression": "",
+          "hide": true,
+          "id": "status2xx",
+          "matchExact": true,
+          "metricName": "HTTPCode_Target_2XX_Count",
+          "namespace": "AWS/ApplicationELB",
+          "period": "300",
+          "refId": "F",
+          "region": "default",
+          "statistics": [
+            "Sum"
+          ]
+        },
+        {
+          "alias": "",
+          "dimensions": {
+            "LoadBalancer": "app/cd-concourse-web/ba5029fcd9b9cde2"
+          },
+          "expression": "",
+          "hide": true,
+          "id": "status3xx",
+          "matchExact": true,
+          "metricName": "HTTPCode_ELB_3XX_Count",
```

Comment:
Similar to 4XX above, I don't think this metric is interesting and we shouldn't track it.

Reply:
yep... agreed ... gone